react/src/components/user/course/CourseChapters.jsx

```
@@ -0,0 +1,134 @@
+                <span>
+                  {sections?.reduce((acc, section) => 
+                    acc + (section.chapters?.reduce((chapterAcc, chapter) => 
+                      chapterAcc + (chapter.lessons?.length || 0), 0
+                    ) || 0), 0
+                  )} lessons
+                </span>
+                <button 
+                  className="btn btn-expand-all"
+                  onClick={toggleAllSections}
+                >
+                  <FontAwesomeIcon icon={allExpanded ? faChevronUp : faChevronDown} />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)